modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -79,7 +79,8 @@ enum { CAP_ANY          = 0,     // autodetect
        CAP_DSHOW        = 700,   // DirectShow (via videoInput)
        CAP_PVAPI        = 800,   // PvAPI, Prosilica GigE SDK
        CAP_OPENNI       = 900,   // OpenNI (for Kinect)
-       CAP_OPENNI_ASUS  = 910,   // OpenNI (for Asus Xtion)
+       CAP_OPENNI_IR    = 910,   // OpenNI (for Kinect IR mode)
```

Comment:
it's actually a bad idea to change numerical values of existing enumeration constants; please, use CAP_OPENNI_IR=920 value instead